acceptance/analyzer_test.go

```diff
@@ -868,6 +874,27 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 						})
 					})
 				})
+
+				when("do have read access to registry", func() {
```

Comment:
```suggestion
				when("no read access", func() {
```

Reply:
Fixed in the following [commit](https://github.com/buildpacks/lifecycle/pull/646/commits/ba8a18c2765ed0444640debec0d3067f5a555ebb)